Meta/Lagom/Tools/CodeGenerators/LibWeb/BindingsGenerator/IDLGenerators.cpp

```diff
@@ -332,6 +332,11 @@ static void emit_includes_for_all_imports(auto& interface, auto& generator, bool
 template<typename ParameterType>
 static void generate_to_string(SourceGenerator& scoped_generator, ParameterType const& parameter, bool variadic, bool optional, Optional<ByteString> const& optional_default_value)
 {
+    if (parameter.type->name() == "USVString")
+        scoped_generator.set("to_string", "to_well_formed_string"sv);
+    else
+        scoped_generator.set("to_string", "to_well_formed_string"sv);
```

Comment:
Shouldn't this be `scoped_generator.set("to_string", "to_string"sv);`? Both of your cases are the same

Reply:
Whoops, yes - unofrtunately i wasnt able to fully test as my laptop was about to die (charging port seems to have broken today). Was working in a reduced test case though w/ all tests pssing.

Just wanted to push up first to get feedback in draft - is this the approach we should be taking for this 🤔 not fully sure yet